ui/hooks/gasFeeInput/useMaxFeePerGasInput.js

```diff
@@ -60,16 +53,9 @@ export function useMaxFeePerGasInput({
 
   const showFiat = useSelector(getShouldShowFiat);
 
-  let initialMaxFeePerGas;
-  if (editGasMode === EDIT_GAS_MODES.SWAPS && supportsEIP1559V2) {
-    initialMaxFeePerGas = swapCustomMaxFeePerGas
-      ? Number(hexWEIToDecGWEI(swapCustomMaxFeePerGas))
-      : null;
-  } else {
-    initialMaxFeePerGas = supportsEIP1559
-      ? getMaxFeePerGasFromTransaction(transaction)
-      : 0;
-  }
+  const initialMaxFeePerGas = supportsEIP1559
```

Comment:
Could you explain why we no longer have to use `swapCustomMaxFeePerGas` when `editGasMode === EDIT_GAS_MODES.SWAPS`?

Reply:
Yep this code I had added in this PR itself, but I was checking this with swaps team and found that this is not needed. Initialisation with high estimate should work for swaps.

I made this change below: https://github.com/MetaMask/metamask-extension/pull/12966/commits/f77752bfd0d563a0a007ac40b98d42f4522c0aaa#diff-d8d61bfbae4f1ef755fb241b4dc5aed4523d7317392c51bd3cbb0bfcd2e55961R70 to fix setting value to `0` and that worked well for initialising swaps correctly.